dev/05_data_core.ipynb

```diff
@@ -405,6 +405,7 @@
    ],
    "source": [
     "test_eq(parent_label(items[0]), '3')\n",
+    "test_eq(parent_label(\"fastai_dev/dev/data/mnist_tiny/train/3/9932.png\"), '3')\n",
```

Comment:
test_eq(parent_label(str(items[0])), '3')

Reply:
Hi @kdorichev 
That definitely looks more concise to me